platform/default/sqlite3.hpp

```diff
@@ -42,27 +48,23 @@ class Database {
 
     explicit operator bool() const;
 
-    static void errorLogCallback(void *arg, const int err, const char *msg);
     void setBusyTimeout(std::chrono::milliseconds);
     void exec(const std::string &sql);
     Statement prepare(const char *query);
 
-    int64_t lastInsertRowid() const;
```

Comment:
Good point - we can add a unit test for verifying that. It's unfortunate that `QSqlDatabase` does not provide direct access for these - so for keeping them in `Database` we'll need a temporary `QSqlQuery` object that makes the calls based on that db.